examples/globe_vector.html

```diff
@@ -49,15 +79,98 @@
             var menuGlobe = new GuiTools('menuDiv');
             menuGlobe.view = globeView;
             // Listen for globe full initialisation event
+            var tooltip = document.getElementById("tooltip");
+            tooltip.style.visibility = 'hidden';
+
+            var mouseDown = 0;
+
+            document.body.onmousedown = function onmousedown() {
+                ++mouseDown;
+            };
+            document.body.onmouseup = function onmouseup() {
+                --mouseDown;
+            };
+
+
             globeView.addEventListener(itowns.GLOBE_VIEW_EVENTS.GLOBE_INITIALIZED, function () {
                 // eslint-disable-next-line no-console
                 console.info('Globe initialized');
                 Promise.all(promises).then(function () {
                     menuGlobe.addImageryLayersGUI(globeView.getLayers(function (l) { return l.type === 'color'; }));
                     menuGlobe.addElevationLayersGUI(globeView.getLayers(function (l) { return l.type === 'elevation'; }));
                     itowns.ColorLayersOrdering.moveLayerToIndex(globeView, 'Ortho', 0);
+                    var layers = globeView.getLayers(function (l) { return l.protocol === 'rasterizer'; });
+
+                    function readPosition(e) {
+                        if (!mouseDown) {
+                            buildToolTip(globeView.controls.pickGeoPosition(e.clientX, e.clientY), e);
+                        } else {
+                            tooltip.style.left = `${e.pageX}px`;
+                            tooltip.style.top = `${e.pageY}px`;
+                        }
+                    }
+
+                    function pickPosition(e) {
+                        buildToolTip(globeView.controls.pickGeoPosition(e.clientX, e.clientY), e);
+                    }
+                    // precision in pixels
+                    var precisionPx = 5;
+                    function buildToolTip(geoCoord, e) {
+                        tooltip.innerHTML = '';
+                        tooltip.style.visibility = 'hidden';
+                        if (geoCoord) {
+                            var visible = false;
+                            var precision = globeView.controls.pixelsToDegrees(precisionPx);
+                            for (var i=0; i < layers.length; i++) {
+                                var layer = layers[i];
+                                    var result = itowns.FeaturesUtils.getFeaturesAtCoordinate(geoCoord, layer.feature, precision);
+                                    var id = 0;
+                                    if (result.points.length || result.lines.length || result.polygons.length) {
+                                        visible = true;
+
+                                        for (var polygon of result.polygons) {
```

Comment:
I don't think you can use `for ... of` in examples (they're not compiled to ES5)